### PR TITLE
[READY] Router external

### PR DESCRIPTION
**Pull request by FelipeSimoes**
Please provide the [GitHub issue(s)](../issues) or short description what the PR does, as well as test URLs where your change can be observed (before and after):

Test URLs:
- Before: https://main--henkel-raqn-guide--hlxsites.hlx.page/
- After: https://router-external--henkel-raqn-guide--hlxsites.hlx.page/test

### CODE REVIEW

File: blocks/accordion/accordion.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import Column from '../column/column.js';
 
 export default class Accordion extends Column {
+  dependencies = ['icon'];
```
Comment (nc-andreashaller) [inline blocks/accordion/accordion.js:4]: left over?

File: blocks/external/external.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+raqn-external {
+  display: none;
+}
+
+raqn-external[inicialized='true'] {
```
Comment (nc-andreashaller) [inline blocks/external/external.css:5]: typo

File: blocks/router/router.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import ComponentBase from '../../scripts/component-base.js';
+
+export default class Router extends ComponentBase {
+  static get observedAttributes() {
+    return ['external'];
+  }
+
+  getPlainUrl(url) {
+    if (url.indexOf('.html') >= 0) {
```
Comment (nc-andreashaller) [inline blocks/router/router.js:9]: typically edge delivery domains do not have an extension, will it still work?

**Review by nc-andreashaller — APPROVED** [review]

File: blocks/accordion/accordion.js (hunk 1 above)
Comment (FelipeSimoes) [reply blocks/accordion/accordion.js:4]: It's a real dependency

File: blocks/router/router.js (hunk 3 above)
Comment (FelipeSimoes) [reply blocks/router/router.js:9]: then it's the last return
```javascript
return `${url}.plain.html`;
```

**Pull request merged**